examples/ldap/variables.tf

```diff
@@ -40,7 +40,7 @@ variable "cores" {
 }
 
 variable "memory" {
-  default     = null
+//  default     = null
```

Comment:
Why is this commented out?  There should probably be a default value for this.  Also, why is an ldap file included in this PR